tasks/discover-pdrs/index.js

```diff
@@ -30,7 +30,9 @@ function discoverPdrs(event) {
       bucket,
       collection,
       provider,
-      config.useList
+      config.useList,
```

Comment:
Please update the schemas for this change and add tests

Reply:
Done